testutil/testnode/full_node.go

```diff
@@ -209,3 +218,15 @@ func DefaultNetwork(t *testing.T, blockTime time.Duration) (cleanup func() error
 		return cleanupGRPC()
 	}, accounts, cctx
 }
+
+func getFreePort() int {
```

Comment:
[note for reviewers]
This is taken from here:

https://github.com/celestiaorg/celestia-node/blob/ef59637c0cf3f9f01ff587b37fa842123e641dc7/core/testing.go#L192-L202